pkgs/applications/networking/instant-messengers/zoom-us/default.nix

```diff
@@ -95,9 +95,28 @@ in mkDerivation {
         cp $icon $out/share/icons/hicolor/$path/apps/us.zoom.Zoom.png
     done
 
-    ln -s $out/share/zoom-us/zoom $out/bin/zoom-us
+
+    # Do the manual Qt wrapping, because existing hook doesn't support our usecase
+    qtWrapperArgs+=(--prefix XDG_DATA_DIRS : "$out/share/zoom-us")
+
+    find "$out/share/zoom-us" ! -type d -executable -print0 | while IFS= read -r -d "" file; do
+      patchelf --print-interpreter "$file" >/dev/null 2>&1 || continue
```

Comment:
doesn't autoPatchelfHook make this unneeded?

Reply:
right! thanks